presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
+* Enforce NOT NULL constraint by checking values for NOT NULL columns.
+* Remove hive-apache dependency on presto-orc.
+* NodeStatusService implementation is now expected to validate & check if a node is allowed to register with DiscoveryNodeManager based on node.location instead of node.id configuration parameter.
+
+JDBC Connector Changes
+______________________
+* JDBC connector allows user to pass username and password from CLI and other clients through use of extra credentials feature.
```

Comment:
```suggestion
* Allow JDBC connectors to accept username and password.
```

Reply:
```
Add support to accept username and password
```

This is the JDBC section - and hence saying "JDBC connectors" is redundant